src/perf/readme.md

```diff
@@ -0,0 +1,152 @@
+exec | `-exec:<lowlat,maxtput,scavenger,realtime>` | The execution profile used for the application.
+pollidle | `-pollidle:<time_us>` | The time, in microseconds, to poll while idle before sleeping (falling back to interrupt-driven IO).
+stats | `-stats:<0,1>` | Prints out statistics at the end of each connection.
+
+# Client
+
+Since tests are client, the client side of secnetperf generally has several arguments passed in. Many of the arguments are also test specific, but the following (overlap with server) do apply to all:
+
+Argument | Usage | Meaning
+--- | --- | ---
+cc | `-cc:<cubic,bbr>` | Congestion control algorithm used.
+cipher | `-cipher:<value>` | Decimal value of 1 or more `QUIC_ALLOWED_CIPHER_SUITE_FLAGS`.
+cpu | `-cpu:<cpu_indexes>` | Comma-separated list of CPUs to run on.
+ecn | `-ecn:<0,1>` | Enables sender-side ECN support.
+exec | `-exec:<lowlat,maxtput,scavenger,realtime>` | The execution profile used for the application.
+pollidle | `-pollidle:<time_us>` | The time, in microseconds, to poll while idle before sleeping (falling back to interrupt-driven IO).
+
+## Throughput Test
+
+To run the throughput test, you must specify the `-test:tput` option, followed by a number of other arguments (most of which are optional).
+
+### Examples
+
+```
+> secnetperf.exe -exec:maxtput -test:tput -target:127.0.0.1 -download:10000000
+```
+
+```
+> secnetperf.exe -test:tput -target:127.0.0.1 -upload:10000 -timed:1 -stats:1
+```
+
+### Required
+
+Argument | Usage | Meaning
+--- | --- | ---
+target,server | `-target:<hostname_or_IP>` | The target server to connect to. May be a hostname or an IP address.
+download | `-download:<####>` | The length of data (or time with `-timed:1` arg) to receive. Mutually exclusive with `-upload` arg.
+upload | `-upload:<####>` | The length of data (or time with `-timed:1` arg) to send. Mutually exclusive with `-download` arg.
+
+### Optional
+
+Argument | Usage | Meaning
+--- | --- | ---
+bind | `-bind:<address>` | Binds to the specified local address.
+cc | `-cc:<cubic,bbr>` | Congestion control algorithm used.
+comp | `-comp:<####>` | The compartment ID to run in. **Windows Only**
+core | `-core:<####>` | The CPU core to use for the main thread.
+port | `-port:<####>` | The UDP port of the server.
+ip | `-ip:<0,4,6> ` | A hint for the resolving the hostname to an IP address.
+cibir | `-cibir:<hex_bytes>` | The well-known CIBIR identifier.
+encrypt | `-encrypt:<0,1>` | Enables/disables encryption.
+sendbuf | `-sendbuf:<0,1>` | Whether to use send buffering.
+pacing | `-pacing:<0,1>` | Whether to use pacing.
+timed | `-timed:<0,1>` | Indicates the `upload` & `download` arg represent time (ms).
+iosize | `-iosize:<####>` | The size of each send request queued.
+tcp | `-tcp:<0,1>` | Indicates TCP/TLS should be used instead of QUIC. **Windows Only**
+stats | `-stats:<0,1>` | Prints out statistics at the end of each connection.
+sstats | `-sstats:<0,1>` | Prints out statistics at the end of each stream.
```

Comment:
Prints out stream level statistics at the end of each stream?

The current wording sounds like `stats` and `sstats` just print the same statistics on different timings.